<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KDE SSH Tunnel Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/icons/kstm.png</normaloff>:/icons/icons/kstm.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>SSH1</string>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/icons/network-disconnect.png</normaloff>
         <normalon>:/icons/icons/network-connect.png</normalon>:/icons/icons/network-disconnect.png</iconset>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_command">
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>229</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_KSTM"/>
   </widget>
   <widget class="QMenu" name="menuTunnel">
    <property name="title">
     <string>Tunnel</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionConnect"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuTunnel"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionConnect"/>
   <addaction name="actionDelete"/>
   <addaction name="actionEdit"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/media-playback-start.png</normaloff>
     <normalon>:/icons/icons/media-playback-stop.png</normalon>:/icons/icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/list-remove.png</normaloff>:/icons/icons/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete tunnel</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/document-properties.png</normaloff>:/icons/icons/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit tunnel</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/list-add.png</normaloff>:/icons/icons/list-add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Add new tunnel</string>
   </property>
  </action>
  <action name="actionAbout_KSTM">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/kstm.png</normaloff>:/icons/icons/kstm.png</iconset>
   </property>
   <property name="text">
    <string>About KSTM</string>
   </property>
   <property name="toolTip">
    <string>Show information about KSTM</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
